src/Aspire.Hosting.AppHost/build/Aspire.Hosting.AppHost.targets

```diff
@@ -246,6 +246,12 @@ namespace Projects%3B
     <MakeDir Directories="$(AspireManifestPublishOutputPath)" />
   </Target>
 
+  <Target Name="__ErrorOnMininumWorkloadVersionMissing" BeforeTargets="PrepareForBuild"
+    Condition="'$(IsAspireHost)' == 'true' and ('$(AspireHostingSDKVersion)' == '' or $([MSBuild]::VersionLessThan('$(AspireHostingSDKVersion)', '8.2.0')))">
+    <Error Code="ASPIRE005"
+      Text="$(MSBuildProjectName) project requires a newer version of the .NET Aspire Workload to work correctly. Please run `dotnet workload update`." />
```

Comment:
> Should this message say what version is required?

I'm not so sure about this one, but I am open to suggestions. The thinking is that this minimum version will change over time, so I'm not sure if we want consistent messages, especially when the fix is to run `dotnet workload update` which you can't really use to select which version to move to (as it always goes to latest).

> Do we need a test for this?

I do think this would be good and I thought about it, but the problem is that it won't be trivial as I don't think we have a way of running tests with an SDK without the workload, or with an older workload, but @radical can correct me if I'm wrong.

Reply:
> > Do we need a test for this?

> I do think this would be good and I thought about it, but the problem is that it won't be trivial as I don't think we have a way of running tests with an SDK without the workload, or with an older workload, but @radical can correct me if I'm wrong.

We do have a way to run tests with a SDK that has the updated manifest from `artifacts`, but does not have the workload installed. I can add that test.

Reply:
> with an older workload
.. this would need some work. Do we want this one?